components/Common/Dropdown/index.module.scss

```
@@ -14,17 +14,30 @@
     > button {
       background: none;
       border: none;
-
       color: var(--color-text-primary);
       cursor: pointer;
       font-size: var(--font-size-body1);
       padding: var(--space-12);
       text-align: center;
       width: 100%;
 
-      &:hover {
-        background-color: var(--color-dropdown-hover);
-      }
-    }
+  &:hover {
+    background-color: var(--color-dropdown-hover);
+   }
+  } 
+ } 
+  &::-webkit-scrollbar {
```

```suggestion

  &::-webkit-scrollbar {
```

It's important to add an empty line after each statement